src/ast/ddl.rs

```diff
@@ -351,6 +351,13 @@ pub enum AlterTableOperation {
     ValidateConstraint {
         name: Ident,
     },
+    /// `SET ( storage_parameter [= value] [, ... ] )`
+    ///
+    /// Note: this is a PostgreSQL-specific operation.
+    /// Please refer to [PostgreSQL documentation](https://www.postgresql.org/docs/current/sql-altertable.html)
```

Comment:
```suggestion
    /// Arbitrary parenthesized `SET` options.
    ///
    /// Example:
    /// ```sql
    /// SET (scale_factor = 0.01, threshold = 500)`
    /// ```
    /// [PostgreSQL](https://www.postgresql.org/docs/current/sql-altertable.html)
```
Can we change to something more generic like the following?